deploy/cloud/modules/ec/variables.tf

```diff
@@ -17,7 +17,7 @@ variable "region" {
 variable "deployment_template" {
   description = "Optional defaults to the CPU optimized template for GCP"
   type        = string
-  default     = "gcp-compute-optimized-v3"
+  default     = "gcp-storage-optimized"
```

Comment:
We always use the default option to deploy the Elastic stack. Since the deployment template `gcp-compute-optimized-v3` has been [deprecated](https://www.elastic.co/guide/en/cloud/current/ec-regions-templates-instances.html), I changed it to the latest one, `gcp-storage-optimized`.